### PR TITLE
design: what screens want

### PR DESCRIPTION
**Pull request by cnhhoang850**
#### What's this PR do?

Publishing "What Screens Want" notes

### CODE REVIEW

**Comment by zlatanpham** [comment]
Hey @cnhhoang850, thanks for your contribution. We will have a look soon.

**Review by monotykamary — CHANGES_REQUESTED** [review]
This was a very inspiring read.

The only thing that needs to be changed is the file extension (`.md` instead of `.d`) with the only minor change is to add some quotations to indicate Frank's famous quote.

File: Design/What Screens Want.d [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+In this regard, when we analyze what makes software unique, it is not the differences in how they are made that matter, it is the user experience (the medium) that is the “message” of software. 
+
+## Screens as the way we understand computation 
+- Because computation is abstract, we need graphic UI to make metaphors "solid".
+- Because our minds are abstract, solid metaphors break down to allow for more attuned experiences once we are adapted to a new level of complexity.
+- The map is not the territory; the abstraction can only serve one purpose.
+- Purposes are normative. 
+- What we want is what screens need, not the other way around.
+## Screens as a medium
+- If something can be anything, it usually becomes everything.
```
[the remark this answers is not in the document]
Comment (cnhhoang850) [reply Design/What Screens Want.d]: Thanks! Added quotations.

**Review by zlatanpham — APPROVED** [review]

**Comment by zlatanpham** [comment]
Hi @cnhhoang850, we've published your note at https://brain.d.foundation/Design/What+Screens+Want. Thank you again for your contribution. Did you join our [Discord server](https://discord.com/invite/Y2vvH9rQE4)? We want to reach out and send a small gift to you :)

**Pull request merged**